src/components/BatteryIndicator/index.tsx

```diff
@@ -65,6 +66,8 @@ const IconContainer = styled.div`
   display: flex;
   align-items: center;
   justify-content: center;
+  // bump the icons above the charge level
```

Comment:
Heads up here – `//` comment syntax is unsupported in CSS (and Styled Components), so we'll want to change this to `/* */`. I use the [StyleLint VSCode extension](https://marketplace.visualstudio.com/items?itemName=stylelint.vscode-stylelint) to check for this kind of stuff. Check it out if you don't have a linter yet!

<img width="650" alt="Screen Shot 2021-06-26 at 10 40 07 AM" src="https://user-images.githubusercontent.com/21055469/123521400-3856f600-d66b-11eb-9084-57d365be3e0a.png">

Reply:
Oh, that's super helpful! Thanks for the tip. :)